keyboards/rico/phoenix_project_no1/readme.md

```diff
@@ -0,0 +1,34 @@
+# Phoenix Project No 1
+
+![Type S](https://github.com/mymakercorner/Phoenix_Project_No1/blob/main/images/Phoenix_Project_No1-TypeS-Back.jpg?raw=true)
+![Type H](https://github.com/mymakercorner/Phoenix_Project_No1/blob/main/images/Phoenix_Project_No1-TypeH-Back.jpg?raw=true)
```

Comment:
Please rehost the images on Imgur or the like.

Reply:
Done